src/tax/vertex-configure-address.md

```diff
@@ -23,23 +23,23 @@ To enable Vertex address validation for the storefront:
 
    This setting enables allows you to configure the address validation settings.
 
-    ![]({% link images/images/config-sales-address-validation.png %}){: .zoom}
-    [_Address Validation settings - enabled_]({% link configuration/sales/address-validation.md %})
+   ![]({% link images/images/config-sales-address-validation.png %}){: .zoom}
+   [_Address Validation settings - enabled_]({% link configuration/sales/address-validation.md %})
 
 1. Verify and accept the default value for **Vertex Address Validation API URL**.
 
    This connects the integration with Vertex Cloud. The value should match the URL displayed in your Vertex Connectors page for the Magento connector in the Address Lookup URL field.
 
 1. If you want to display a message when the address is correctly verified, set **Confirmation Message when no action is needed**  to `Enable`.
 
-    The default is to disable this function so that a message is displayed only if the address does not match a validated address with a prompt to correct it.
+   The default is to disable this function so that a message is displayed only if the address does not match a validated address with a prompt to correct it.
```

Comment:
```suggestion
   By default, this function is disabled so that a message is displayed only if the address does not match a validated address with a prompt to correct it.
```